node/service/src/lib.rs

```diff
@@ -583,10 +584,9 @@ impl ExecuteWithClient for RevertConsensus {
 
 	fn execute_with_client<Client, Api, Backend>(self, client: Arc<Client>) -> Self::Output
 	where
-		<Api as sp_api::ApiExt<Block>>::StateBackend: sp_api::StateBackend<BlakeTwo256>,
-		Backend: sc_client_api::Backend<Block> + 'static,
+		Backend: sc_client_api::Backend<Block>,
 		Backend::State: sp_api::StateBackend<BlakeTwo256>,
-		Api: RuntimeApiCollection<StateBackend = Backend::State>,
```

Comment:
This change comes from substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)